bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -364,6 +407,18 @@ while numpy.any(fnlouder == 0):
         f['foreground_h%s/time1' % h_iterations] = all_trigs.time1[fore_locs]
         f['foreground_h%s/time2' % h_iterations] = all_trigs.time2[fore_locs]
 
+    if is_bkg_inc == True:
+        louder_foreground = fnlouder
```

Comment:
Don't you also need to recalculate this by calling ``coinc.calculate_n_louder()`` again?

Reply:
^^good point, I think I was drifting towards that while statement.

`coinc.calculate_n_louder()` is calculated a few lines above in this code, I believe

Reply:
this line sits down stream of https://github.com/ligo-cbc/pycbc/blob/master/bin/hdfcoinc/pycbc_coinc_statmap#L331-L332

But I'll put it closer to the top so that the code is easier to read.